packages/core/admin/admin/src/content-manager/components/BlocksEditor/BlocksInput/index.js

```diff
@@ -62,11 +62,23 @@ const BlocksInput = ({ disabled }) => {
     }
   };
 
+  const handleBackspaceEvent = (event) => {
+    const selectedNode = editor.children[editor.selection.anchor.path[0]];
+    const selectedBlock = Object.values(blocks).find((block) => block.matchNode(selectedNode));
+
+    if (selectedBlock.value.type === 'list') {
+      selectedBlock.handleBackspaceKey(editor, event);
```

Comment:
You can just check if selectedBlock.handleBackspaceKey exists before calling it in order to avoid having block-specific logic in here